.env.example

```diff
@@ -21,3 +23,10 @@ PAGE_LIST=
 # BLOG_NAME=
 # BLOG_TITLE=
 # BLOG_SUBTITLE=
+
+# TIMELINE_LIST=
+# DATE_ID=
```

Comment:
Could it be `TIMELINE_DATE_ID` to namespace it within the plugin, so different plugins could add different custom_fields if needed?

Reply:
Sorted